packages/block-library/src/embed/variations.js

```diff
@@ -360,6 +361,14 @@ const variations = [
 		patterns: [ /^https?:\/\/(www\.)?wolframcloud\.com\/obj\/.+/i ],
 		attributes: { providerNameSlug: 'wolfram-cloud', responsive: true },
 	},
+	{
+		name: 'bluesky',
+		title: 'Bluesky',
+		icon: embedBlueskyIcon,
+		description: __( 'Embed a Bluesky post.' ),
+		patterns: [ /^https?:\/\/(www\.)?bsky\.app\/obj\/.+/i ],
```

Comment:
```suggestion
		patterns: [ /^https?:\/\/bsky\.app\/profile\/.+\/post\/.+/i ],
```

Looking at [the core changeset](https://core.trac.wordpress.org/changeset/58091), isn't this the pattern?

cc @swissspidy

Reply:
@t-hamano Thank you for catching that! I've updated it to match the correct pattern now. Appreciate your attention to detail!